### DISCUSSION TITLE
Add debug visualization of inertial parameters (debug render inertial boxes)

### DISCUSSION THREAD

**Discussion by erwincoumans** [bulletphysics/bullet3#3497] [General]
It is a useful debugging tool to show the inertial boxes, around the center of mass.

See also gazebo:
"Every link should now display a purple box with green axes. The center of each box is aligned with the specified center of mass of its link. The sizes and orientations of the boxes correspond to unit-mass boxes with the same inertial behavior as their corresponding links. "
(from http://gazebosim.org/tutorials?tut=inertia&cat=build_robot)

**Comment by scpeters** [comment]
The box size is computed in [InertiaVisual.cc](https://bitbucket.org/osrf/gazebo/src/533f2914646453e3dc090d1795ca3e8924bcc683/gazebo/rendering/InertiaVisual.cc#InertiaVisual.cc-52). I think it's a useful tool too. :)
